provider/bidengine/pricing.go

```diff
@@ -293,6 +295,10 @@ func (ssp shellScriptPricing) CalculatePrice(ctx context.Context, gspec *dtypes.
 	outputBuf := &bytes.Buffer{}
 	cmd.Stdout = outputBuf
 
+	subprocEnv := os.Environ()
+	subprocEnv = append(subprocEnv, fmt.Sprintf("AKASH_OWNER=%s", owner))
```

Comment:
can we use prefix from config instead of direct `AKASH`?

Reply:
does subprocess also support multiple addresses in `AKASH_OWNER`

Reply:
I could make this configurable, but I guess why would it need to be? I was trying to keep the same format like I used here https://github.com/ovrclk/akash/blob/master/provider/cluster/kube/builder.go#L318